docs/general/getting-started/popular-dapps.md

```diff
@@ -0,0 +1,9 @@
+---
+sidebar_label: '🤖 Popular Dapps'
+sidebar_position: 4
+description: Some interesting dapps built on LUKSO
+---
+
+# Popular Dapps and Projects
+
+BLAH BLAH DAPPY DAPP
```

Comment:
I don't think we can have this kind of sentences here.
Suggest putting some temporary callout here to say this page is under construction.

```
:::info
This page is under construction
:::
```

Reply:
Whoops! This wasn't supposed to go live.